config/default_config.xml

```diff
@@ -33,7 +33,7 @@
   #     write_DEBUG        = (True|False) used to write lots of DEBUG info to the DEBUG file ("aaajunk.out")
   #===========================================================
 -->   

```

Comment:
The options in this file should be moved to conf.ini.default for those which are actually used (I feel like some of them do no need to be kept).

Reply:
The parameter write_DEBUG is not used anymore.

Level of printing is now handle via logger.